kitsune/questions/jinja2/questions/includes/aaq_macros.html

```diff
@@ -103,19 +116,27 @@ <h2 class="card--title has-bottom-margin">{{ _('Ask the Community') }}</h2>
         href="{{ next_step }}"
         data-event-category="link click"
         data-event-action="{{ location }}"
-        data-event-label="aaq widget">{{ _('Ask Now') }}</a>
+        data-event-label="aaq widget">
+        {% if aaq_context.has_subscriptions %}
+          {{ _('Contact Support') }}
+        {% else %}
+          {{ _('Ask Now') }}
+        {% endif %}
+      </a>
     </div>
   {% endif %}
 {%- endmacro %}
 
-{% macro explore_solutions(product, search_box, featured, topics, request) -%}
+{% macro explore_solutions(product, search_box, featured, topics, request, zendesk_form=True) -%}
```

Comment:
instead of passing another flag here, can't we use `product.has_subscriptions` in the `if` statement?